frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1938,18 +1939,33 @@ end
 function ReaderHighlight:onMoveHighlightIndicator(args)
     if self.view.visible_area and self._current_indicator_pos then
         local dx, dy, quick_move = unpack(args)
-        local step_distance = self.view.visible_area.w / 5 -- quick move distance: fifth of visible_area
-        local y_step_distance = self.view.visible_area.h / 5
-        if step_distance > y_step_distance then
-            -- take the smaller, make all direction move distance much predictable
-            step_distance = y_step_distance
-        end
-        if not quick_move then
-            step_distance = step_distance / 4 -- twentieth of visible_area
-        end
+        local quickly_move_distance_dx = self.view.visible_area.w / 5 -- quick move distance: fifth of visible_area
+        local quickly_move_distance_dy = self.view.visible_area.h / 5
```

Comment:
May be change this "quickly" to "quick", just so it reads better english and consistent ?